app/javascript/styles/mastodon/components.scss

```diff
@@ -1052,6 +1052,29 @@
       }
     }
   }
+
+  &-unlisted .status__relative-time::before,
+  &-private .status__relative-time::before,
+  &-direct .status__relative-time::before {
+      display: inline-block;
+      text-align: center;
+      width: 14px;
+      height: 14px;
+      font: 14px/1 FontAwesome;
+      margin-right: 4px;
+  }
+
+  &-unlisted .status__relative-time::before {
+      content: "\f09C";
+  }
+
+  &-private .status__relative-time::before {
+      content: "\F023";
+  }
+
+  &-direct .status__relative-time::before {
+      content: "\F0E0";
+  }
```

Comment:
I'm a bit uneasy relying on “internal” FontAwesome values like this.

Reply:
I agree! Please do not do this, this is not accessible by any means. The content of pseudo elements is being read by screenreaders and can lead to the confusion, instead you might want to place span with icon (`<i>`), inside or before the relative time in the code of status component, and provide `aria-label` to it describing privacy level for people, who unable to see the icon.